### ISSUE TITLE
Add use_issue_template or perhaps a `use_template_*` family?

### ISSUE THREAD

**Issue by njtierney** [r-lib/usethis#335]
[Issue templates](https://github.com/tidyverse/dplyr/blob/master/ISSUE_TEMPLATE.md) Are really nifty! 

It would be great to have a function like:

`use_issue_template` that would provide a similar template to the one linked above.

I'm happy to submit a PR for this, but before I do so I just wanted to check if this would be useful?

**Comment by njtierney** [comment]
Ah, thanks for that @jennybc :)

I actually didn't see `use_tidy_issue_template()`! I tried digging around the github issues but I didn't think to look inside the package properly! Sorry about that.

`use_tidy_issue_template()` definitely does the trick, but are there templates for pull requests?

Also, I currently have these things living in a .github folder - it looks like that isn't needed now to make the templates appear?

**Comment by jennybc** [comment]
> but are there templates for pull requests?

No. Our experience with issue templates is that people ignore them, so I think that tempered our enthusiasm for creating more templates. But it would be nice, so I don't oppose covering this.

> Also, I currently have these things living in a .github folder - it looks like that isn't needed now to make the templates appear?

I think usethis is a bit schizophrenic wrt this right now. For tidyverse-specific stuff, we **have** switched to a model that puts all this below `.github/`. But there's probably room for rationalization or flexibility around this for helping other people get these files where they want them.
